esp/src/src-react/util/metricArchive.ts

```diff
@@ -107,7 +107,7 @@ export class Archive {
                 }
                 this.sourcePath_Metrics[definition.filePath].add(metric.id);
             });
-            return prev + totalTime ?? 0;
+            return prev + totalTime;
```

Comment:
Could this result in something unexpected like `NaN` getting returned (doing arithmetic with null or undefined)?